files/69-available-updates-begin.sh.amzn2

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+# Copyright (C) 2024 Amazon.com, Inc. or its affiliates.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the
+# License.
+
+package_updates=$(LANG=C timeout 30s /usr/bin/yum \
+  --debuglevel 2 \
+  --security check-update 2>/dev/null \
+  | grep -P '(?<! 0 packages) available$')
```

Comment:
This is copied from `/etc/update-motd.d/30-banner` that's inherited from the base AL2 AMI.
```
#!/bin/bash

# Possible summaries include:
# No packages needed for security; %d packages available
# %d package(s) needed[ (+%d related)] for security, out of %d available
# There are [[%d security update(s)[ out of ]%d total update(s)]] available
LANG=C timeout 30s /usr/bin/yum \
    --debuglevel 2 \
    --security check-update 2>/dev/null \
        | grep -P '(?<! 0 packages) available$' \
    && echo 'Run "sudo yum update" to apply all updates.'
```

Reply:
We an possibly add a summary comment of what is being done here, referencing base AL2 updates